data/presets/climbing/crag.json

```diff
@@ -0,0 +1,15 @@
+{
+    "icon": "temaki-climbing",
+    "fields": [
+        "name"
+    ],
+    "geometry": [
+        "relation",
+        "point"
```

Comment:
What about 1.6k ways? https://taginfo.openstreetmap.org/tags/climbing=crag#overview

Reply:
Personally, I dislike the practice of tagging cliffs with `climbing=crag`. To me the concept of a crag is distinct from the cliff itself, and mixing the two feels like it breaks with the [one feature, one OSM element](https://wiki.openstreetmap.org/wiki/One_feature,_one_OSM_element) guideline.

However, this pattern seems so well established that I don't feel it's my role to argue against it at this point. I added "line" as a possible geometry.